ai-agent/agent.py

```
@@ -0,0 +1,81 @@
+    """
+    agent = await create_agent(
+        agent_id=agent_id,
+        agent_name=agent_name,
+        instructions=instructions
+    )
+    await join_call(agent, call_type, call_id)
+
+
+if __name__ == "__main__":
+    # CLI mode for testing
+    cli(AgentLauncher(create_agent=create_agent, join_call=join_call))
```

Ruff (0.14.13)

Undefined name `cli`

(F821)